strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RegenerateNugetChecksumCronJobTestIT.java

```diff
@@ -136,86 +116,4 @@ public void testRegenerateNugetChecksumInStorage(@NugetRepository(storageId = ST
-                                    artifactNupkgPath,
-                                    null,
-                                    null,
-                                    additionalProperties);
-    }
-
-    private void testRegenerateNugetChecksum(Repository repository,
-                                             Path artifactNupkgPath,
-                                             String cronJobConfigStorageId,
-                                             String cronJobConfigRepositoryId,
-                                             Map<String, String> cronJobConfigProperties)
-            throws Exception
-    {
-        final UUID jobKey = expectedJobKey;
-        final String jobName = expectedJobName;
-
-        RootRepositoryPath repositoryPath = repositoryPathResolver.resolve(repository);
-
-        Path nupkgSha512Path = artifactNupkgPath.resolveSibling(artifactNupkgPath.getFileName() + ".sha512");
-        deleteIfExists(nupkgSha512Path);
-        assertThat(Files.notExists(nupkgSha512Path))
-                .as("The checksum file for nupkg artifact exist!")
-                .isTrue();
-
-        NugetArtifactCoordinates coordinates = (NugetArtifactCoordinates) RepositoryFiles.readCoordinates(
-                (RepositoryPath) artifactNupkgPath.normalize());
-        coordinates.setType("nuspec");
-        Path nuspecSha512Path = repositoryPath.resolve(coordinates.toPath() + ".sha512");
-        deleteIfExists(nuspecSha512Path);
-        assertThat(Files.notExists(nuspecSha512Path))
-                .as("The checksum file for nuspec artifact exist!")
-                .isTrue();
-
-        List<Path> resultList = new ArrayList<>();
-        jobManager.registerExecutionListener(jobKey.toString(), (jobKey1, statusExecuted) -> {
-            if (!StringUtils.equals(jobKey1, jobKey.toString()) || !statusExecuted)
-            {
-                return;
-            }
-            resultList.add(nupkgSha512Path);
-            resultList.add(nuspecSha512Path);
-        });
-
-        addCronJobConfig(jobKey,
-                         jobName,
-                         RegenerateChecksumCronJob.class,
-                         cronJobConfigStorageId,
-                         cronJobConfigRepositoryId,
-                         properties -> properties.putAll(cronJobConfigProperties));
-
-        await().atMost(EVENT_TIMEOUT_SECONDS, TimeUnit.SECONDS).untilTrue(receivedExpectedEvent());
-
-        assertThat(resultList).hasSize(2);
-        resultList.forEach(ThrowingConsumer.unchecked(path -> {
-            assertThat(Files.exists(path))
-                    .as("The checksum file " + path.toString() + " doesn't exist!")
-                    .isTrue();
-            assertThat(Files.size(path) > 0)
-                    .as("The checksum file is empty!")
-                    .isTrue();
-        }));
-
-        assertThat(Files.size(artifactNupkgPath)).isBetween(BYTE_SIZE, DEFAULT_BYTE_SIZE);
-    }
-}
+}
```

Comment:
Cosmetic: Please, add new line at the end of the file. Thanks! :)

Reply:
fixed

Reply:
Thanks!